docs/rules/escape-user-inputs-in-sql-queries.md

```diff
@@ -0,0 +1,170 @@
+---
+title: Escape User Inputs in SQL Queries
+ruleId: EscapeUserInputsInSQLQueries
+since: 3.17.0
+minJavaVersion: 1.1
+remediationCost: 10
+links:
+    - displayName: "Prepared Statements (with Parameterized Queries)"
+      url: "https://cheatsheetseries.owasp.org/cheatsheets/SQL_Injection_Prevention_Cheat_Sheet.html#defense-option-4-escaping-all-user-supplied-input"
+    
+description:
+    Rule description.
+tags: ["Security"]
+---
+
+# Escape User Inputs in SQL Queries
+
+[[toc]]
+
+## Properties
+
+<RuleProperties />
+
+## Description
+
+This rule prevents Injection by escaping user input before putting it in a SQL-query. It can be applied when an ORACLE-database management system is used. 
+
+## Benefits
+
+Prevents SQL injections.
+
+## Requirement & Tags
+
+::: warning Requirements
+Java 1.1
+:::
+
+::: tip Tags
+<TagLinks />
+:::
+
+## Code Changes
+
+
+### Using Statement execute
+
+__Pre__
+```java
+HttpServletRequest req = getRequest();
+String query = "SELECT first_name FROM employee WHERE department_id ='" +  req.getParameter("departmentId") + "' ORDER BY last_name";
+Statement statement = connection.createStatement();
+statement.execute(query);
+ResultSet resultSet = statement.getResultSet();
+```
+
+__Post__
+```java
+HttpServletRequest req = getRequest();
+Codec<Character> ORACLE_CODEC = new OracleCodec();
+String query = "SELECT first_name FROM employee WHERE department_id ='" +  ESAPI.encoder().encodeForSQL(ORACLE_CODEC, req.getParameter("departmentId")) + "' ORDER BY last_name";
+Statement statement = connection.createStatement();
+statement.execute(query);
+ResultSet resultSet = statement.getResultSet();
+```
+
+__Pre__
```

Comment:
I think the example above is enough to create an idea. We don't want to overwhelm the reader with examples.

Reply:
OK in all examples I will only take the example with the invocation of ```getParameter```